src/main/webapp/js/view-model.js

```diff
@@ -223,9 +223,8 @@ const CoverageChartGenerator = function ($) {
         // TODO: maybe it would make sense to render only visible charts
         function initializeCharts() {
             renderTrendChart();
-
             viewProxy.getOverview(function (t) {
-                createOverview(t.responseObject(), 'coverage-overview');
+                createOverview(t.responseObject(), 'coverage-overview', t.responseJSON.coverageCoveredText, t.responseJSON.coverageMissedText);
```

Comment:
We may only need to pass `t.responseObject()` as an argument, since it's almost same as ` t.responseJSON` when use.